script.js

```diff
@@ -779,8 +844,9 @@ function sendBookingData(packageID) {
 }
 
 // update booking status
-function updateStatus(travelerID, statusID) {
+function updateStatus(travelerID, bookingID, statusID) {
     var travelerID = travelerID;
+    var bookingID = bookingID;
     var statusID = statusID;
```

Comment:
This variable assignment operation is useless. You can directly usetravelerID, bookingID, and statusID variables.